packages/product-components/src/components/CoinLogo/CoinLogo.stories.tsx

```diff
@@ -37,5 +37,11 @@ export const CoinLogo: StoryObj<CoinLogoProps> = {
                 type: 'select',
             },
         },
+        type: {
+            options: ['coin', 'network', 'badge'],
```

Comment:
```suggestion
            options: ['token', 'network', 'tokenNetworkBadge'],
```
I would improve naming. I am not sure with this though `tokenNetworkBadge` any ideas?

Reply:
What about `layerNetworkBadge`?

Reply:
I dont know
tokenWithNetworkBadge? 😀

Reply:
`tokenWithNetwork` ship-it

Reply:
done